tensorboard/webapp/tb_wrapper/tb_wrapper_component.ts

```diff
@@ -37,4 +46,27 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
   ],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class TensorBoardWrapperComponent {}
+export class TensorBoardWrapperComponent {
```

Comment:
It's unclear to me that this is the best spot to put this when considering other flavors of TensorBoard but I'm not sure where would be better and thus am open to suggestions.

Reply:
I am also not sure this is the best spot. I would have to do some exploring to see if I can find a better spot. But, maybe someone else will have a good suggestion so I will wait on other reviews before digging further.

Reply:
How about:
* Create new FeatureFlagModalTriggerComponent that contains this logic.
* Put <feature-flag-modal-trigger> in AppContainer: https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/app_container.ng.html

By putting it in AppContainer you would get the modal for free in all routes (in theory -- if Standalone TensorBoard actually contained other routes). But, more importantly, it would be a model for both TensorBoard.corp and TensorBoard.dev to also follow (they have their own AppContainers) and they do have multiple routes.

Reply:
And, to be clear, by "Put in AppContainer" I mean modify app_container.ng.html to refer to <feature-flag-modal-trigger> component.

Reply:
I believe I've made the change you suggested.